src/test/java/com/sfuentes/ticket/TicketTest.java

```diff
@@ -0,0 +1,75 @@
+package com.sfuentes.ticket;
+
+import com.sfuentes.grocery.Client;
+import com.sfuentes.grocery.Grocery;
+import com.sfuentes.grocery.Item;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TicketTest {
+
+  @Test
+  void ticket_decimalResults_returnChange() {
```

Comment:
Please follow the naming convention for tests:
MethodName_StateUnderTest_ExpectedBehavior
In the state under test your are writing a result.

Reply:
fixed